packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -110,6 +110,27 @@ module.exports = {
       }
     ],
     loaders: [
+      // Default loader: load all assets from appSrc that are not already
+      // handled by other loaders with the url loader.
+      // Note: Any changes to the other loader tests that apply to appSrc
+      // need to be reflected here too.
+      // E.g., when adding a loader for new supported file extensions
+      // in appSrc, we need to add the new extension to this list too.
+      //
+      // "file" loader makes sure those assets get served by WebpackDevServer.
+      // When you `import` an asset, you get its (virtual) filename.
+      // In production, they would get copied to the `build` folder.
+      // "url" loader works like "file" loader except that it embeds assets
+      // smaller than specified limit in bytes as data URLs to avoid requests.
+      {
+        test: /\.(?!(js|jsx|css|json|html)(\?.*)?$)[^\.]+$/,
```

Comment:
The idea was to include exactly the same regexes used in those other loaders, so each file is either handled by one of them or handled by this loader. The JS, JSON and CSS loader configs we have don't match query strings in their regexes, so there's no need to match them here either. I think the whole loader config could be as simple as:
```js
{
  test: /./,
  loader: 'url',
  query: {
    limit: 10000,
    name: 'static/media/[name].[hash:8].[ext]'
  },
  exclude: [
    /\.(js|jsx)$/,
    /\.css$/,
    /\.json$/
  ]
}
```